cli/src/main/resources/application.yml

```diff
@@ -2,7 +2,7 @@ spring:
   jmx:
     enabled: false
   profiles:
-    active: receiver,ssl
+    active: receiver,ssl,amqp-adapter-cli
```

Comment:
My understanding is that this property defines the profiles to be active by default. FMPOV it doesn't make sense to activate receiver together with adapter client, does it?

Reply:
actually, i was wondering about this... i want to completely remove the `amqp-adapter-cli` profile and activate it using the `--spring.profiles.active` switch. WDYT?

Reply:
>  i want to completely remove the amqp-adapter-cli profile and activate it using the --spring.profiles.active switch. WDYT?

FMPOV you simply need to remove `amqp-adapter-cli` from this line and then you will be able to do exactly what you want to do, i.e. using the `--spring.profiles.active` switch ...